.github/workflows/release.yml

```diff
@@ -126,7 +153,7 @@ jobs:
           path: Docs.zip*
 
   upload-release:
-    needs: build
+    needs: [build, create_draft_release]
```

Comment:
I've moved the dependency since I can't find anywhere where `build` depends on the draft release.